frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/details-card.tsx

```diff
@@ -53,6 +63,21 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
     InfrastructureModel,
     'cluster',
   );
+  const [ocsSubscription, ocsSubscriptionLoaded, ocsSubscriptionError] = useK8sWatchResource<
```

Comment:
```suggestion
  const [subscription, subscriptionLoaded, subscriptionError] = useK8sWatchResource<
```
You are technically queriying for all subscriptions. Could we limit this fetching to perhaps only `ocs` subscription?

Reply:
We already have the subscription work used in this file.